modules/nf-core/nanofilt/main.nf

```diff
@@ -0,0 +1,53 @@
+process NANOFILT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanofilt:2.8.0--py_0':
+        'biocontainers/nanofilt:2.8.0--py_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    path  summary_file
+
+    output:
+    tuple val(meta), path("*.fastq.gz"), emit: filtreads
+    path "versions.yml"                , emit: versions
+    path "*.log"                       , optional: true, emit: log_file
```

Comment:
```suggestion
    path "*.log"                       , optional: true, emit: log_file
    path "versions.yml"                , emit: versions
```
Unwritten convention is to have the versions at the bottom. Typically the order of channels is the order files are written out.